rasa/cli/data.py

```diff
@@ -146,8 +149,13 @@ def _build_training_data_importer(args: argparse.Namespace) -> "TrainingDataImpo
         args.config, "config", DEFAULT_CONFIG_PATH, none_is_valid=True
     )
 
+    # Exit the validation if the domain path is invalid
+    domain = rasa.cli.utils.get_validated_path(
+        args.domain, "domain", None, none_is_valid=False
```

Comment:
Could we use the DEFAULT_DOMAIN_PATH instead of None, this way you also don't need to edit an existing test?

Reply:
It should probably catch it as an error (`The path 'not-existing-domain.yml' does not exist.`) instead of defaulting to `domain.yml` when we do the `data validate` command, for `train` it defaults to `domain.yml` where it's `Encountered empty domain during validation.` error.

Reply:
Unless the project you create in the test has a `domain.yml` file created, maybe remove that 😄